src/main/java/frc/robot/Robot.java

```diff
@@ -91,7 +91,9 @@ public void teleopInit() {
 
   /** This function is called periodically during operator control. */
   @Override
-  public void teleopPeriodic() {}
+  public void teleopPeriodic() {
+    m_robotContainer.setSpeedPercent();
```

Comment:
Variables: No under_scores (except constants).

Reply:
I didn't write that variable, Stevon mentioned that in the other PR request. It's part of the REV swerve code.

Reply:
If we are going to change the underscores here we might as well change it everywhere in the code (See [#8](https://github.com/BCHRobotics/Trojans2024/pull/8#discussion_r1454209174))

Reply:
Yes, will create ticket for this, after I double check the standards.

Reply:
> I didn't write that variable, Stevon mentioned that in the other PR request. It's part of the REV swerve code.

Do you have a link to the code reference?

Reply:
Opened issue for this #12